client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -0,0 +1,150 @@
+			},
+			icon: sshIcon,
+		},
+		{
+			name: 'domains',
+			label: __( 'Domains' ),
+			searchLabel: __( 'domains' ),
+			context: 'Managing domains',
+			callback: setStateCallback( 'domains' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/domains/manage/${ site.slug }/dns/${ site.slug }` );
+				},
+				filter: ( site: SiteExcerptData ) =>
+					isCustomDomain( site.slug ) && ! isNotAtomicJetpack( site ),
+			},
+			icon: domainsIcon,
+		},
+		{
+			name: 'hostingConfiguration',
+			label: __( 'Hosting Configuration' ),
+			searchLabel: __( 'hosting' ),
+			context: 'Configuring hosting settings',
+			callback: setStateCallback( 'hostingConfiguration' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/hosting-config/${ site.slug }` );
+				},
+				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+			},
+			icon: hostingConfigurationIcon,
+		},
+		{
+			name: 'newPage',
+			label: __( 'New Page' ),
+			searchLabel: __( 'create page' ),
+			context: 'Adding a new page',
+			callback: setStateCallback( 'newPage' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/page/${ site.slug }` );
+				},
+			},
+			icon: newPageIcon,
+		},
+		{
+			name: 'newPost',
+			label: __( 'New Post' ),
+			searchLabel: __( 'create post' ),
+			context: 'Adding a new post',
+			callback: setStateCallback( 'newPost' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/post/${ site.slug }` );
+				},
+			},
+			icon: newPostIcon,
+		},
+		{
+			name: 'publish',
+			label: __( 'Publish' ),
+			searchLabel: __( 'publish content' ),
+			context: 'Publishing the content',
+			callback: setStateCallback( 'publish' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/settings/general/${ site.slug }#site-privacy-settings` );
+				},
+				filter: ( site: SiteExcerptData ) => site?.launch_status === 'unlaunched',
+			},
+			icon: publishIcon,
+		},
+		{
+			name: 'view',
+			label: __( 'View site' ),
+			searchLabel: __( 'view public site frontend' ),
+			context: 'Publishing the content',
+			callback: setStateCallback( 'view' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( site.URL );
+				},
+			},
+			icon: viewSitehIcon,
+		},
+		{
+			name: 'addNewSite',
+			label: __( 'Add New Site' ),
+			searchLabel: __( 'new site' ),
+			context: 'Adding a new website',
+			callback: ( { close }: { close: () => void } ) => {
+				close();
+				navigate( createSiteUrl );
+			},
+			icon: addNewSiteIcon,
+			separator: true,
+		},
```

Comment:
It would be great if we can simplify the list of commands for the initial prototype.

What do you think about adding only these?

- Add new site | wp.com/start
- Open my profile | wp.com/me
- Open account settings | wp.com/me/account
- Open my purchases | wp.com/me/account
- Manage domains | wp.com/me/purchases